tdrs-backend/docs/api/stts.md

```diff
@@ -0,0 +1,60 @@
+# STTs
```

Comment:
The API documentation is terrific!

Reply:
Much appreciated , however we're still working towards an OpenAPI supported schema to move away from these manual updates

Reply:
I love it!